app/assets/stylesheets/budget.css

```diff
@@ -0,0 +1,239 @@
+* {
+  color: #526138;
```

Comment:
I would suggest to use variables for the colours. Not sure if you are using the conventions Elle suggested, but I think it's very useful!

Reply:
I just looked at yours and thought I would try variables :)